actions/update-plugins

```diff
@@ -0,0 +1,28 @@
+#!/usr/local/sbin/charm-env python3
+
+from charmhelpers.core.hookenv import (
+    function_fail,
+    function_set,
+)
+
+from charms.layer.jenkins.api import Api
+
+
+def update_plugins():
+    api = Api()
+    api.check_update_center()
+    plugins = api.get_updatable_plugins()
+    if len(plugins) > 0:
+        api.update_plugins()
+        api.restart()
+        function_set({'output': "Updated plugins: {}".format(plugins),
+                      'outcome': 'Success'})
+    else:
+        function_set({'output': "No plugins available to be updated.",
```

Comment:
Are we saying here that there are no plugins that need updating, or that we couldn't find any plugins? The message seems a little ambiguous to me as worded currently.

Reply:
Rephrased it.

Reply:
c8feeab